background.html

```diff
@@ -434,6 +434,10 @@ <h3 class='header'>{{ newIdentity }}</h3>
           <label for='theme-setting-android'>Android</label>
         </div>
         <div>
+          <input type='radio' name='theme' id='theme-setting-android-dark' value='android-dark'>
+          <label for='theme-setting-android'>{{ themeAndroidDark }}</label>
```

Comment:
Bug: clicking this label selects the non-dark theme.
Fix: s/android/android-dark/